.github/config.yml

```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Layer5-celebration.png)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/images/welcome/meshery-welcome-logo-light-text-side.svg">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/images/welcome/meshery-welcome-logo-dark-text-side.svg">
```

Comment:
Better use the raw URLs for images, also rename the images to Meshery-celebration-Dark.svg and Meshery-celebration-Light.svg to keep the uniformity of name conventions

Reply:
Also, the dimensions of the images are larger than needed, use these 
![Meshery-celebration-Dark](https://github.com/meshery/meshery-traefik-mesh/assets/92504666/cdaa15c8-7508-4b1f-87e8-9d883f25ea94)
![Meshery-celebration-Light](https://github.com/meshery/meshery-traefik-mesh/assets/92504666/84e9948d-f2db-4bf2-a7e1-c143444ba768)

Reply:
Thanks for these reviews. Let me fix it and confirm with you

Reply:
Thanks @thisiskaransgit You can review the changes again. In case of any adjustment, please let me know